### PR TITLE
Feauture/expanding buttons

### PR DESCRIPTION
**Pull request by MsweliP**
Restricting the graph and changing UI

### CODE REVIEW

File: waterlog/src/Components/Map.js [hunk 1]
```diff
@@ -100,8 +104,8 @@ class MapComponent extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      lat: -25.783,
-      lng: 28.337,
+      lat: -25.783425,
+			lng: 28.336046,
```
Comment (FredJEhlers) [inline waterlog/src/Components/Map.js:110]: fix indentation and split line 137 over multiple lines

File: waterlog/src/Components/Map.js [hunk 2]
```diff
@@ -129,8 +133,9 @@ class MapComponent extends Component {
     return (
       <div className="map-main-div">
         <div className="map-tile-div">
-          <Map center={position} zoom={this.state.zoom} zoomControl={false} style={{height:this.props.height}}>
-            {(() => {
+			
+     <Map center={position} maxBounds={[[-25.784510,28.334360],[-25.782110,28.338325]]} zoom={this.state.zoom} zoomControl={false} style={{height:this.props.height}}>
```
Comment (Lavhe) [inline waterlog/src/Components/Map.js]: Why not create a variable for maxBounds as well?

**Review by Lavhe — CHANGES_REQUESTED** [review]

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/Map.js (hunk 2 above)
Comment (MsweliP) [reply waterlog/src/Components/Map.js]: Done , please re-check again

File: waterlog/src/Components/Map.js [hunk 3]
```diff
@@ -134,7 +136,7 @@ class MapComponent extends Component {
       <div className="map-main-div">
         <div className="map-tile-div">
 			
-     <Map center={position} maxBounds={[[-25.784510,28.334360],[-25.782110,28.338325]]} zoom={this.state.zoom} zoomControl={false} style={{height:this.props.height}}>
+     <Map center={position} maxBounds={[southWest,northEast]} zoom={this.state.zoom} zoomControl={false} style={{height:this.props.height}}>
```
Comment (FredJEhlers) [inline waterlog/src/Components/Map.js]: Split over two lines

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/Map.js (hunk 3 above)
Comment (Lavhe) [reply waterlog/src/Components/Map.js]: 👍 cool

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**